internal/commands/push/command.go

```diff
@@ -444,3 +444,40 @@ func deployDraftAndWait(ui terminal.UI, realmClient realm.Client, remote appRemo
 	ui.Print(terminal.NewTextLog("Deployment complete"))
 	return nil
 }
+
+func importDependenciesAndWait(ui terminal.UI, realmClient realm.Client, remote appRemote, uploadPath string) error {
+	err := realmClient.ImportDependencies(remote.GroupID, remote.AppID, uploadPath)
+	if err != nil {
+		return err
+	}
+
+	s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+	s.Suffix = " Installing dependencies..."
+
+	waitForInstallation := func() error {
+		s.Start()
+		defer s.Stop()
+
+		status := realm.DependenciesStatusCreated
+		for status == realm.DependenciesStatusCreated {
+			time.Sleep(time.Second)
+			install, err := realmClient.DependenciesInstallation(remote.GroupID, remote.AppID)
+			if err != nil {
+				return err
+			}
+			s.Suffix = fmt.Sprintf(" %s...", install.StatusMessage)
```

Comment:
At the moment, StatusMessage just ends up being `processing` so the spinner that displays is like `[] processing...`. I'm not sure if this is intentional but does not seem user friendly since other spinners have messages like `[] Installing dependencies...` so perhaps capitalization / more descriptive message would be nice to have?